UPGRADE.md

```diff
@@ -2,6 +2,74 @@
 
 ## dev-develop
 
+### Twig-Extensions
+
+Following Twig-Functions has changed the name (new prefix for sulu functions):
+
+| Before                    | Now                           |
+|---------------------------|-------------------------------|
+| `resolve_user`            | `sulu_resolve_user`           |
+| `content_path`            | `sulu_content_path`           |
+| `content_root_path`       | `sulu_content_root_path`      |
+| `get_type`                | `sulu_get_type`               |
+| `needs_add_button`        | `sulu_needs_add_button`       |
+| `get_params`              | `sulu_get_params`             |
+| `parameter_to_select`     | `sulu_parameter_to_select`    |
+| `parameter_to_key_value`  | `sulu_parameter_to_key_value` |
+| `content_load`            | `sulu_content_load`           |
+| `content_load_parent`     | `sulu_content_load_parent`    |
+| `get_media_url`           | `sulu_get_media_url`          |
+| `meta_alternate`          | `sulu_meta_alternate`         |
+| `meta_seo`                | `sulu_meta_seo`               |
+| `navigation_root_flat`    | `sulu_navigation_root_flat`   |
+| `navigation_root_tree`    | `sulu_navigation_root_tree`   |
+| `navigation_flat`         | `sulu_navigation_flat`        |
+| `navigation_tree`         | `sulu_navigation_tree`        |
+| `breadcrumb`              | `sulu_breadcrumb`             |
+| `sitemap_url`             | `sulu_sitemap_url`            |
+| `sitemap`                 | `sulu_sitemap`                |
+| `snippet_load`            | `sulu_snippet_load`           |
+
+To automatically update this name you can run the following script. If your themes are not in the ClientWebsiteBundle
+you have to change the folder in the second line.
+
+```
+#!/usr/bin/env bash
+TWIGS=($(find ./src/Client/Bundle/WebsiteBundle/Resources/themes -type f -iname "*.twig"))
+
+NAMES[0]="resolve_user"
+NAMES[1]="content_path"
+NAMES[2]="content_root_path"
+NAMES[3]="get_type"
+NAMES[4]="needs_add_button"
+NAMES[5]="get_params"
+NAMES[6]="parameter_to_select"
+NAMES[7]="parameter_to_key_value"
+NAMES[8]="content_load"
+NAMES[9]="content_load_parent"
+NAMES[10]="get_media_url"
+NAMES[11]="meta_alternate"
+NAMES[12]="meta_seo"
+NAMES[13]="navigation_root_flat"
+NAMES[14]="navigation_root_tree"
+NAMES[15]="navigation_flat"
+NAMES[16]="navigation_tree"
+NAMES[17]="breadcrumb"
+NAMES[18]="sitemap_url"
+NAMES[19]="sitemap"
+NAMES[20]="snippet_load"
```

Comment:
Nice. I think you could simplify this by having just a space delimited string, e.g. `for name in "one two three four"`, but not important.